doc/04-Upgrading.md

```diff
@@ -0,0 +1,21 @@
+# Upgrading Icinga DB <a id="upgrading"></a>
+
+## Upgrading to Icinga DB RC2
+
+Icinga DB RC2 is a complete rewrite of RC1. Because of this, a lot has changed in the Redis and database schema, which
```

Comment:
"rewrite of RC1" sounds strange to me, IMHO "rewrite compared to RC1" would better describe it.